Makefile

```diff
@@ -333,6 +341,7 @@ MATH_SRC +=\
 	$(SOURCEDIR)/Math/GPUWatcher.cu \
 	$(SOURCEDIR)/Math/GPURNGHandle.cu \
 	$(SOURCEDIR)/Math/MatrixQuantizerGPU.cu \
+        $(SOURCEDIR)/Math/NcclComm.cpp \
```

Comment:
Minor: could you use spaces here, as on previous line?
[Our Makefile has somewhat mixed space styles, which we'll fix; but let's try to be locally consistent.